python-sdk/tests/experiment/test_async_trace_isolation.py

```
@@ -0,0 +1,318 @@
+@pytest.fixture(scope="module", autouse=True)
+def setup_langwatch():
+    os.environ["LANGWATCH_API_KEY"] = "test-key-for-async-tracing"
+    langwatch._api_key = "test-key-for-async-tracing"
+    langwatch._endpoint = "http://localhost:5560"
+    try:
+        langwatch.setup()
+    except Exception:
+        # Tracer may already be installed by a previous test module; reusing
+        # the existing provider is fine for these tests since trace_ids are
+        # what we assert, not exporter state.
+        pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)